### DISCUSSION TITLE
Data inheritance with Class Inheritance

### DISCUSSION THREAD

**Discussion by cancan101** [pimcore/pimcore#16941] [Q&A / Support] [answered]
Can "Data inheritance" be used between two different classes that are "different", but in which one _inherits_ from the other?

For example if I have classes `Vehicle` and `Car` (where `Car` extends `Vehicle`), can I place a `Car` object under a `Vehicle` and have data fields inherited?

I have seen the note in the docs:
>Data inheritance between different classes is not supported.

However I wasn't sure if the child class in this class is [considered "a" parent](https://en.wikipedia.org/wiki/Liskov_substitution_principle).

**Comment by dpfaffenbauer** [comment, marked as answer]
Not possible, works only for the same class.
